pkg/minikube/assets/addons.go

```diff
@@ -346,7 +348,7 @@ var Addons = map[string]*Addon{
 			vmpath.GuestAddonsDir,
 			"registry-creds-rc.yaml",
```

Comment:
how did we decide we maintain the registry creds ? I believe the images used in this addon are not pushed-contrlled by google.

so I would say, anything Google does not maintain the "Image" for it and has push access should not be listed as Google.

for example gcp-auth is maintained by googler, and is pushed by google. we dont know who pushes the registry-cred image

Reply:
I was mostly looking at the git blame for the maintainers, I think I updated it all correctly. Updated PR description so please look it over!